deploy/cloud-ingress-operator-configuration/sshd/config.yaml

```diff
@@ -0,0 +1,9 @@
+deploymentMode: "SelectorSyncSet"
```

Comment:
Does this need to be `Upsert` I guess not since we're creating it.

Reply:
No it doesn't since we're just creating here.

Reply:
And the sync set in CIO is already [upsert](https://github.com/openshift/cloud-ingress-operator/blob/master/hack/olm-registry/olm-artifacts-template.yaml#L50) in preparation for the removal